real_time/arduino_src/lib_avr/rbserialmessages/rbserialmessages.cpp

```diff
@@ -18,13 +18,10 @@
 
 // Public //////////////////////////////////////////////////////////////////////
 
-RBSerialMessages::RBSerialMessages() 
-{
-}
+RBSerialMessages::RBSerialMessages() {} //constructor function
```

Comment:
Based on how cpp is, I don't think it's necessary to clarify that these empty functions are constructors